server/src/concepts/offer/useCases/updateOffer.js

```diff
@@ -0,0 +1,19 @@
+import { updateOffer } from "../repositories/commands";
+import sendEmail from "../../../services/mail";
+//import Offer from "../model/Offer";
+
```

Comment:
I realised I have similar validation in controllers so maybe second one is useless

Reply:
you should check here if offer exists. It is good approach.